### ISSUE TITLE
Console output doesn't print non-ascii prediction.

### ISSUE THREAD

**Issue by MPThLee** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#246]
**Describe the bug**
The Console output doesn't print non-ascii(like utf-8/cjk characters) prediction.
For Example, Prediction with name `ASDF ああああ ASDF` only prints like `ASDF  ASDF` in console output.
However, File log is ok. It does write prediction name as `ASDF ああああ ASDF`.

**To Reproduce**
1. Set channel which prediction held on (in utf-8 characters, whatever)
2. Compare the predictions, console output, file log, and twitch web ui.

**Expected behavior**
Print all name correctly.

**Desktop (please complete the following information):**
 - OS: Debian buster AArch64
 - Python version: 3.7.3
 - VENV: with buster's `virtualenv` with wrapper `pipenv`

**Comment by RakSrinaNa** [comment]
Are you sure it isn't an issue with your setup/virtual env?

I have a raspberry with debian aarch64 and I see Japanese characters fine in the console output?

**Comment by MPThLee** [comment]
No. It seems like it's not my environment issue. I just tried with this code for test the internal logger on same environment.
```py
from TwitchChannelPointsMiner.logger import LoggerSettings, configure_loggers
import logging
logger_settings = LoggerSettings()
logger = logging.getLogger(__name__)
configure_loggers("test", logger_settings)

logger.info("1. there's test character in left, with four 'a': ああああ")
logger.info("にぃ。ああああ")
logger.info("3. hello ああああ")
logger.info("四. it started with kanji")
logger.info("5. testing korean(한국어) and chinese(中文)")
```
There's no issue on print non-ascii characters.
But, on the real application(`run.py` as described in readme.), it doesn't printed at all. This applies korean and chinese too.

I will set debug mode for testing this. seems like there's issue with server communication.

**Comment by MPThLee** [comment]
Ok, I found the what it causes. The `emoji` removal function has a bug. seems it removes unrelated characters too.
I just set `logger_settings = LoggerSettings(emoji=False)` and I can reproduce the error.
![image](https://user-images.githubusercontent.com/26970092/129453586-9ceb3db0-d7d0-4501-a19c-1f9c5c3843d5.png)

**Comment by RakSrinaNa** [comment]
Can do a PR to remove the invalid characters from the list

**Comment by Tkd-Alex** [comment]
Uhm :thinking:  We wrote the emoji removal function for Windows user, If we remove those characters from the list a Windows user with `emoji=false` will continue to see artifacts on the console

**Comment by RakSrinaNa** [comment]
I am on windows and I don't see any artifacts. Depends on the setup used.

And in that case it should probably be split into two different things, kinda like `emoji=on/off` and `asciiOnly=on/off`. And each one do what they say (maybe ascii implicitly means emoji=off though as emojis are not ascii).

**Comment by MPThLee** [comment]
> Uhm 🤔 We wrote the emoji removal function for Windows user, If we remove those characters from the list a Windows user with `emoji=false` will continue to see artifacts on the console

The CJK characters are not emoji. (Well, Someone uses it as emoji /shrug). 
The range of characters of removal emoji in code is too wide. It includes characters other than emojis. ([Here's Unicode's full-list of their emoji sequence.](https://unicode.org/Public/emoji/13.1/emoji-sequences.txt)). 
I mean, Those range list contains emoji, but also contains many other non-emoji characters.

The current code is more like remove non-ascii. not just emoji
